package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@roadiehq/plugin-github-pull-requests",
```

Comment:
I'm conflicted about how to name these plugins. I want to make it obvious that they are backstage plugins, but  `backstage-plugin-github-pull-requests` is a very long name. The Spotify team get away with this because `@backstage` makes it obvious that it's a backstage plugin.

Which do you prefer? `"@roadiehq/plugin-github-pull-requests"` or `"@roadiehq/backstage-plugin-github-pull-requests"`?

Reply:
True that, I think "@roadiehq/backstage-plugin-github-pull-requests" is most descriptive and we shouldn't worry about this name being long.

Reply:
Also the name of the repo already fits the "@roadiehq/backstage-plugin-github-pull-requests" package name

Reply:
👍